### PR TITLE
fix: add <AsserELfeki>

### LINKED ISSUE #1753: [PROFILE ADDITION] <AsserElfeki>
### Profile screenshot

image

### PR DESCRIPTION
**Pull request by AsserElfeki**
resolves #1753

## Checklist for profile addition

- [ ] Name the PR `fix: add <your-username>`.
- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/AsserELfeki.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "hhttps://user-images.githubusercontent.com/66274223/222985800-f94ad181-7030-4f69-96df-83da43221e96.png",
```
Comment (Panquesito7) [inline profiles/AsserELfeki.json:2]: Wrong link.
```suggestion
  "image": "https://user-images.githubusercontent.com/66274223/222985800-f94ad181-7030-4f69-96df-83da43221e96.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by AsserElfeki** [comment]
thank you :)

**Comment by AsserElfeki** [comment]
will it be merged ?

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**